openapi-to-har.js

```diff
@@ -483,7 +486,7 @@ const parseParametersToQuery = function (openApi, parameters, values) {
     }
     if (typeof param.in !== 'undefined' && param.in.toLowerCase() === 'query') {
       // param.name is a safe key, because the spec defines
-      // that name MUST be unique
+      // that name MUST be uniques
```

Comment:
My keyboard is breaking at my explicit saves (Cmd-S) have been leaving stray `s` characters everywhere. Grrr